docs/breakingchanges-0.63.md

```diff
@@ -0,0 +1,95 @@
+
```

Comment:
@NickGerleman: What would be the appropriate place for this?

Reply:
That's a really good question 🙂Definitely of the opinion that I'd rather we document this somewhere even if we don't have a great place yet.

We haven't been providing anything quite as comprehensive as this, since we've had tons and tons of template changes in each version. We want to describe breaking changes on the blog during release, which is a separate Git repo `react-native-sampleapps`. We'll need to remember to surface this then.

@kikisaints have any ideas here?

Reply:
Re: tooling, there's discussion of ramping up react-native-upgrade-helper support. There hasn't been active investment in it yet though.